docs/advanced/aml.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""This example demonstrates how to use ray to submit a job to Azure ML using ray-on-aml."""
```

Comment:
we typically create files in rst format in docs folder.

Reply:
It is not possible to use aml in doctest.

Reply:
Why did you put this script in the docs folder instead of examples folder?